src/components/Pool/PairView/UserLiquidity/UserLiquidity.tsx

```diff
@@ -70,14 +71,22 @@ export function UserLiquidity({ pair }: UserLiquidityProps) {
             title={t('userLiquidity.poolTokens')}
             value={userPoolBalance ? userPoolBalance.toSignificant(4) : '0'}
           />
-          <ValueWithLabel
-            title={t('userLiquidity.pooledToken', { token: currency0?.symbol })}
-            value={token0Deposited ? token0Deposited.toSignificant(6) : '0'}
-          />
-          <ValueWithLabel
-            title={t('userLiquidity.pooledToken', { token: currency1?.symbol })}
-            value={token1Deposited ? token1Deposited.toSignificant(6) : '0'}
-          />
+          <ValueWithLabel title={t('userLiquidity.pooledToken', { token: currency0?.symbol })}>
+            <Flex alignItems="center">
+              <Box mr="6px">
+                <CurrencyLogo size="14px" currency={currency0} />
+              </Box>
+              {token0Deposited ? token0Deposited.toSignificant(6) : '0'}
+            </Flex>
+          </ValueWithLabel>
+          <ValueWithLabel title={t('userLiquidity.pooledToken', { token: currency1?.symbol })}>
+            <Flex alignItems="center">
+              <Box mr="6px">
+                <CurrencyLogo size="14px" currency={currency1} />
+              </Box>
+              {token1Deposited ? token1Deposited.toSignificant(6) : '0'}
+            </Flex>
+          </ValueWithLabel>
```

Comment:
Proper style is missing. Value text style different from the other values.

Reply:
Didn't understand what you mean with proper style.

Reply:
Got it, thanks!

Reply:
Fixed on 1e92cbc7b569298ee2ac5b719bd5a82a2f868fad